build-tools/packages/build-cli/src/handlers/checkFunctions.ts

```diff
@@ -446,6 +437,64 @@ export const checkPolicy: StateHandlerFunction = async (
 	return true;
 };
 
+/**
+ * Checks that a release branch exists.
+ *
+ * @param state - The current state machine state.
+ * @param machine - The state machine.
+ * @param testMode - Set to true to run function in test mode.
+ * @param log - A logger that the function can use for logging.
+ * @param data - An object with handler-specific contextual data.
+ * @returns True if the state was handled; false otherwise.
+ */
+export const checkAssertTagging: StateHandlerFunction = async (
+	state: MachineState,
+	machine: Machine<unknown>,
+	testMode: boolean,
+	log: CommandLogger,
+	data: FluidReleaseStateHandlerData,
+): Promise<boolean> => {
+	if (testMode) return true;
+
+	const { context, releaseGroup, shouldCheckPolicy } = data;
+	assert(context !== undefined, "Context is undefined.");
+	assert(releaseGroup !== undefined, "Release group is undefined.");
+
+	if (shouldCheckPolicy === true) {
+		if (getPolicyRunDefault(releaseGroup, context.originalBranchName)) {
+			log.warning(
+				`Assert tagging for ${releaseGroup} is not expected on the ${context.originalBranchName} branch! Make sure you know what you are doing.`,
+			);
+		}
+
+		// Run script from the root of the repo
```

Comment:
if we run this on the repo root, then it only runs on the client release group, right?

We do we support requesting tunning it on other release groups? Shouldn't we just no-op it on groups where it doesn't make sense (no work to do for it), or make it an error instead of running it on client even if the current group is not client?

Reply:
policy-check is scoped to the path that it's run in. Since we have multiple folders at the root that represent the client release group, we can't easily scope it to just the client. What I'm trying to do here is simulate what the repo-policy-check CI job does.

Reply:
I think that description would be better placed in a comment here than in the PR discussion.

Reply:
Put this info in the comments in the latest iteration.